mote/projects/base/library/browser/patterns/atoms/image/data.yaml

```diff
@@ -1,2 +1,2 @@
-attrs:
-    src: foo.png
+#attrs:
```

Comment:
I commented out all this to ensure that dummy data does not leak through to production.